js/app.js

```diff
@@ -13,36 +13,36 @@ Picture.currentSet = [];
 Picture.lastSet = [];
 
 //Constructor creates picture object
-function Picture(name, filePath) {
+function Picture(name, filePath, index) {
   this.name = name;
   this.filePath = filePath;
+  this.index = index;
```

Comment:
If you want to keep track of the index, it might be better to calculate the index instead of having it as a parameter. Since you are pushing the object to the end of `Picture.allPics`, the index should be at `Picture.allPics.length`.